share/picongpu/dockerfiles/ubuntu-1604/Dockerfile

```diff
@@ -1,80 +1,76 @@
-FROM       nvidia/cuda:8.0-devel
+FROM       nvidia/cuda:9.0-base
 MAINTAINER Axel Huebl <a.huebl@hzdr.de>
 
 # docker and image environment
 ENV        DEBIAN_FRONTEND=noninteractive \
-           PICONGPU_BRANCH="0.3.0" \
-           HOME=/home/picongpu \
-           PATH=/home/picongpu/src/spack/bin:$PATH
+           FORCE_UNSAFE_CONFIGURE=1 \
+           SPACK_ROOT=/usr/local \
+           SPACK_EXTRA_REPO=/usr/local/share/spack-repo \
+           PIC_PACKAGE='picongpu@develop+isaac backend=cuda'
 
 # install minimal spack dependencies
 #   - adds gfortran for spack's openmpi package
-#   - adds the standard editors for users: vim, nano, emacs
+#   - adds the standard editors for users: vim, nano
+#     (excluded: emacs-nox pulls up to 90MB in)
 #   - adds build tools which are NOT linked in the final app
+#   - selected cuda toolkit components
+# note: do you need more cuda toolkit components?
+# check:   apt-get update && apt-cache search cuda && apt-cache search nvcc
 RUN        apt-get update && \
            apt-get install -y --no-install-recommends \
               autoconf \
               build-essential \
               ca-certificates \
               coreutils \
+              cuda-cublas-dev-$CUDA_PKG_VERSION \
```

Comment:
why do you pull `cublas` for picongpu?

Reply:
there is a include in `cuda_memtest.h` of the third party tool  `cuda_memtest`. This is a bug because there are no blas routines used.